locales-cs-CZ.xml

```diff
@@ -251,6 +251,7 @@
     </term>
 
     <!-- VERB ROLE FORMS -->
+    <term name="container-author" form="verb"/>
```

Comment:
Anyway, the point I was trying to make is that it would be safer to keep the `<term></term>` format, just in case parsing is different in some of the CSL processors, although I'm not sure this is a real concern.